webpack/app/galleria/theme.js

```diff
@@ -12,12 +12,14 @@ var rm_button = function(data) {
   return "<i class='tile-remove-item' data-uuid='" + uuid + "'><span class='text'>remove</span></i>";
 };
 
-(function($) {
+!(function($) {
```

Comment:
bump version in:

https://github.com/collective/collective.cover/blob/master/webpack/package.json#L3

Reply:
I bumped it from 2.1.2 to 2.1.3, is that ok?

Reply:
With the python package updated to 3.0.0, I think it's better to put the same version here. So we leave space if someone wants to do something on a 2.x branch.

I made it. Thanks!